lib/updatehub/updatehub.c

```diff
@@ -431,6 +431,9 @@ static void install_update_cb(void)
 	}
 
 	ctx.code_status = UPDATEHUB_OK;
+#if defined(CONFIG_WATCHDOG)
+	updatehub_watchdog_kick();
```

Comment:
Should you add the default impl for it?

Reply:
@otavio I provided default impl. now, but checkpatch still flags the same.